realm/src/androidTest/java/io/realm/internal/test/TestHelper.java

```diff
@@ -20,24 +20,24 @@
      * @param o
      * @return
      */
-    public static ColumnType getColumnType(Object o){
+    public static RealmFieldType getColumnType(Object o){
```

Comment:
Right now we still use the convention that the internal API talks about columns/tables etc. Changing this would break that convention. 

I know that renaming `ColumnType` to `RealmFieldType` is also a step in that direction, but that is motivated by it becoming public.

So I would rather keep the renaming of other internal API methods out of the scope of this PR as it might cascade into a much bigger renaming.